tests/Feature/CaddyApplicationLabelGenerationTest.php

```
@@ -0,0 +1,38 @@
+    $server->settings->update(['generate_exact_labels' => true]);
+    $destination = StandaloneDocker::query()->where('server_id', $server->id)->firstOrFail();
+    $application = Application::factory()->create([
+        'environment_id' => $environment->id,
+        'destination_id' => $destination->id,
+        'destination_type' => $destination->getMorphClass(),
+        'fqdn' => 'https://example.com',
+        'redirect' => 'both',
+        'is_http_basic_auth_enabled' => false,
+    ]);
+
+    $labels = generateLabelsApplication($application);
```

PHPStan (2.2.7)

[warning] Parameter `#1` $application of function generateLabelsApplication expects App\Models\Application, Illuminate\Database\Eloquent\Collection<int, Illuminate\Database\Eloquent\Model>|Illuminate\Database\Eloquent\Model given.


(argument.type)